main.go

```diff
@@ -5,12 +5,14 @@ import (
 	"net/http"
 	_ "net/http/pprof" // import for side effects
 
+	"encoding/json"
```

Comment:
I recommend you use goimports as your formatter so this gets formatted properly.